src/app/home/index/index.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="home-container">
+  <a class="home-redirect" href="/posts">Go to posts!</a>
```

Comment:
Never use href in angular, use [routerLink]="'/posts'"

Reply:
ok. Done